client/wallet/webconnector.go

```diff
@@ -100,6 +108,11 @@ func (webConnector WebConnector) UnspentOutputs(addresses ...address.Address) (u
 
 // SendTransaction sends a new transaction to the network.
 func (webConnector WebConnector) SendTransaction(tx *devnetvm.Transaction) (err error) {
+	rateSetterInfo, err := webConnector.client.RateSetter()
+	if err != nil {
+		return err
+	}
+	time.Sleep(rateSetterInfo.Estimate)
```

Comment:
Couldn't the client expose such a functionality? And you just call it from here?